packages/crab-netting/src/CrabNetting.sol

```diff
@@ -7,49 +9,126 @@ contract CrabNetting {
     address usdc;
     address crab;
 
-    mapping(address => uint256) usd_balance;
-    mapping(address => uint256) crab_balance;
+    mapping(address => uint256) public usd_balance;
+    mapping(address => uint256) public crab_balance;
```

Comment:
Not a big deal, we should be consistent with variable naming. Should be either camel case or snake case